neopia/neosoco.py

```diff
@@ -425,6 +425,22 @@ def motor_move(self, direction='forward'):
         else:
             raise TypeError
         Runner.wait(100) # Since broadcast from controller is per 100ms
+        
+    def motor_stop(self, which_motor: str):
+        if isinstance(which_motor, str):
+            if which_motor.lower() =='right':
+                self.write(Neosoco.RIGHT_MOTOR, 0)
+            elif which_motor.lower() =='left':
+                self.write(Neosoco.LEFT_MOTOR, 0)
+            elif which_motor.lower() =='both':
+                self.write(Neosoco.LEFT_MOTOR, 0)
+                self.write(Neosoco.RIGHT_MOTOR, 0)
+            else:
+                Util.print_error('Wrong value of motor')
+                raise ValueError
+        else:
+            raise TypeError
+        Runner.wait(100) # Since brocast from cotroller is per 100ms

```

Comment:
I mentioned before as my comment(https://github.com/roboticsware/pylib_neobot/pull/5#discussion_r942486719) on the previous PR. Is the wait function mandatory`?